docs/data/sources/google-tag-manager-client.md

```diff
@@ -369,3 +339,33 @@ If you hardcode the value in your tag, the input will be forced into a string ty
 This video tutorial walks through the implementation basics. 
 
 <script src="https://fast.wistia.com/embed/medias/n337njhoot.jsonp" async></script><script src="https://fast.wistia.com/assets/external/E-v1.js" async></script><div class="wistia_responsive_padding" style="padding:56.25% 0 0 0;position:relative;"><div class="wistia_responsive_wrapper" style="height:100%;left:0;position:absolute;top:0;width:100%;"><div class="wistia_embed wistia_async_n337njhoot videoFoam=true" style="height:100%;position:relative;width:100%"><div class="wistia_swatch" style="height:100%;left:0;opacity:0;overflow:hidden;position:absolute;top:0;transition:opacity 200ms;width:100%;"><img src="https://fast.wistia.com/embed/medias/n337njhoot/swatch" style="filter:blur(5px);height:100%;object-fit:contain;width:100%;" alt="" aria-hidden="true" onload="this.parentNode.style.opacity=1;" /></div></div></div></div>
+
+## Breaking changes checklist
+
+### Page view event_type and event_properties
+
+The new template changes the default page view events to include `[Amplitude]` prefixes. If you want to continue using the older page view events check `Use legacy page view properties`. See full details in table. 
+
+???Breaking change "Page View Tracking"
+    | <div class="big-column">Before</div>  | Current |
+    | --- | --- |
+    | <ul><li>event_type: `Page View`</li><li>event_properties: `page_location`, `page_path`, `page_title`. `page_url`</li></ul> | <ul><li>event_type:  `[Amplitude] Page Viewed`</li><li>event properties: `[Amplitude] Page Domain`, `[Amplitude] Page Location`, `[Amplitude] Page Path`, `[Amplitude] Page Title`, `[Amplitude] Page URL`</li></ul> |
+
+### Subdomain attribution tracking 
+
+Traffic from one subdomain to another (ie analytics.amplitude.com to experiment.amplitude.com) is not tracked by default. If you want to exclude the attribution tracking on `location.hostname`, but not other subdomains, add the value of `location.hostname` in the exclude referral section. See full details in table. 
+
+???Breaking change "Attribution Tracking"
+    | <div class="big-column">Before</div>  | Current |
+    | --- | --- | --- | --- |
+    | Track attribution of all subdomains. | Excludes all subdomains of the same root domain as referrer | 
+
+### User agent parser
+
+The new template changes the way to parse the device related info which might effect the value of `event.os_name`, `event.os_version`, `event.device_model`, `event.device_manufacturer` and related properties. If you want to continue using the older way to parse user agent, check `Enable client side user agent enrichment `. See full details in table.
```

Comment:
Should we map the check box `Enable client side user agent enrichment` to the new version?

Reply:
`(Legacy) Enable client side user agent enrichment`